sources/osgPlugins/ReaderWriterGLTF.js

```diff
@@ -39,15 +45,27 @@ var ReaderWriterGLTF = function() {
     this._basicAnimationManager = undefined;
     this._visitedNodes = undefined;
     this._animatedNodes = undefined;
-    this._skeletons = undefined;
     this._bones = undefined;
     this._skeletonToInfluenceMap = undefined;
     this._inputImgReader = undefined;
     this._localPath = '';
 
+    this._extensions = [];
+
+    this._nodeAnimationTypes = undefined;
     this.init();
 };
 
+function base64ToArrayBuffer(base64) {
```

Comment:
could be also static in FileHelper?

Reply:
It was but because it's only used in gltf, I did not want to move it yet